templates/tools/dockerfile/clang_update.include

```diff
@@ -0,0 +1,32 @@
+#=================
+# Update clang to a version with improved tsan
+
+RUN apt-get update && apt-get -y install python cmake && apt-get clean
+
+RUN git clone -n -b release_38 http://llvm.org/git/llvm.git && ${'\\'}
```

Comment:
What do you think of doing a checkout by a specific timestamp? That way we just have to change the desired timestamp each time we update.